src/purchaseMovies.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+const purchaseMovies = async (moviesInsideCart) => {
+
+    const url = "https://api.mocklets.com/mock68075/";
+    const config = { headers: {"X-Mocklets-PublicKey" : "txmovies", "X-Mocklets-Checksum" : "830c7cd4a70be6540a4898441ca02951"}};
+    let moviesToBuy ={data:{movies:[]}};
+
+    moviesInsideCart.map((movie) => {
```

Comment:
you don't need a map, just use a forEach. 
https://codeburst.io/javascript-map-vs-foreach-f38111822c0f

moviesInsideCart.forEach((movie) => {
        moviesToBuy.data.movies.push(movie.id);
    })